faq/manage-cluster-faq.md

```diff
@@ -35,16 +37,20 @@ By default, TiDB/PD/TiKV outputs standard error in the logs. If a log file is sp
 
 ### How to safely stop TiDB?
 
-Kill all the services using `kill` directly. The components of TiDB will do `graceful shutdown`.
+- A load balancer is running (recommended): Stop the load balancer and then run the kill command. Then TiDB waits for a period specified by [graceful-wait-before-shutdown](/tidb-configuration-file.md#graceful-wait-before-shutdown-new-in-v50) until all sessions are terminated. Then TiDB stops running.
+
+- No load balancer is running: Run the kill command. The TiDB components are gracefully stopped.
```

Comment:
This is ambiguous, since there is a shell command `kill` as well as `kill` in MySQL.

There is also a `SHUTDOWN` sql statement, which might be best to recommend since users might not know which `KILL` level to use etc. See: https://docs.pingcap.com/tidb/stable/sql-statement-shutdown

Reply:
@easonn7 PTAL, thanks.

Reply:
I agree with Morgan which we should recommend shutdown and tiup cluster command instead of [kill].
But it seems that we do not have a shutdown [all] option to stop the entire cluster.